lti_consumer/utils.py

```diff
@@ -26,22 +26,36 @@ def _(text):
     return text
 
 
-def get_lms_base():
+def get_lti_api_base():
     """
-    Returns LMS base url to be used as issuer on OAuth2 flows
-    and in various LTI URLs. For local testing it is often necessary
-    to override the normal LMS base with a proxy such as ngrok, use
-    the setting LTI_LMS_BASE_URL_OVERRIDE in your LMS settings if
-    necessary.
+    Returns base url to be used as issuer on OAuth2 flows
+    and in various LTI API calls.
 
     TODO: This needs to be improved and account for Open edX sites and
     organizations.
     One possible improvement is to use `contentstore.get_lms_link_for_item`
     and strip the base domain name.
     """
-    if hasattr(settings, 'LTI_LMS_BASE_URL_OVERRIDE'):
-        return settings.LTI_LMS_BASE_URL_OVERRIDE
+    if hasattr(settings, 'LTI_API_BASE_URL'):
+        return settings.LTI_API_BASE_URL
+    elif hasattr(settings, 'LTI_VIEW_BASE_URL'):
```

Comment:
it's interesting that API defaults to VIEW if available but not the other way around

Reply:
I was thinking of this as view url is required and api url is an optional override but the naming I chose doesn't really make that clear.